db/models/articles.model.js

```diff
@@ -18,6 +18,20 @@ function selectAllArticles(){
     })
 } 
 
+function updateArticle(article_id, update){
+    return db.query('SELECT * FROM articles WHERE article_id=$1', [article_id])
+    .then((result) => {
+        if (result.rows.length === 0){
+             return Promise.reject({status: 404, msg: 'article does not exist'})
+        }
+        if (typeof update !== 'number'){
+            next(err)
+        }
```

Comment:
I think this is not required as the the catch block on the controller will catch the error thrown by PSQL in the event that id is not a number.